docs/data/joy/customization/theme-global-variants/theme-global-variants.md

```diff
@@ -0,0 +1,125 @@
+# Global variant tokens
+
+<p class="description">Learn how to customize the global variant tokens.</p>
+
+## Prerequisite
+
+- Read the [global variant concept](/joy-ui/core-features/global-variant/) page.
+- Read the [theme tokens](/joy-ui/customization/theme-tokens/) page.
+- (optional) Completed [the tutorial](/joy-ui/getting-started/tutorial/).
+
+## How it works
+
+Joy has four built-in global variants, `plain` `outlined` `soft` and `solid`. The styles generated for each variant comes from the global variant tokens inside the theme palette.
+
+The tokens are composed of three things:
```

Comment:
Probably using something like this is more descriptive:

`[variant][state][CSS property]`